src/Microsoft.DotNet.ProjectModel.Server/InternalModels/ProjectContextSnapshot.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.DotNet.ProjectModel.Server.Helpers;
+using Microsoft.DotNet.ProjectModel.Server.Models;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.ProjectModel.Server.InternalModels
```

Comment:
to be consistent with `ProjectSnapshot` rename the namespace to `Microsoft.DotNet.ProjectModel.Server`